
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264588399 -3600
# Node ID 30e510081e5da36c6a1ce875ecdef13ea08c4a5e
# Parent  e6c311469ee1929bd422b90fa422fab0c12077eb
libv4l: Fixup some license headers

From: Hans de Goede <hdegoede@redhat.com>

Fix some copy and paste errors in license headers

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/include/libv4l1.h	Wed Jan 27 11:30:06 2010 +0100
+++ b/v4l2-apps/libv4l/include/libv4l1.h	Wed Jan 27 11:33:19 2010 +0100
@@ -9,7 +9,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software
--- a/v4l2-apps/libv4l/include/libv4l2.h	Wed Jan 27 11:30:06 2010 +0100
+++ b/v4l2-apps/libv4l/include/libv4l2.h	Wed Jan 27 11:33:19 2010 +0100
@@ -9,7 +9,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software
--- a/v4l2-apps/libv4l/include/libv4lconvert.h	Wed Jan 27 11:30:06 2010 +0100
+++ b/v4l2-apps/libv4l/include/libv4lconvert.h	Wed Jan 27 11:33:19 2010 +0100
@@ -9,7 +9,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software
--- a/v4l2-apps/libv4l/libv4l1/libv4l1.c	Wed Jan 27 11:30:06 2010 +0100
+++ b/v4l2-apps/libv4l/libv4l1/libv4l1.c	Wed Jan 27 11:33:19 2010 +0100
@@ -11,7 +11,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software
--- a/v4l2-apps/libv4l/libv4lconvert/processing/libv4lprocessing.h	Wed Jan 27 11:30:06 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/processing/libv4lprocessing.h	Wed Jan 27 11:33:19 2010 +0100
@@ -11,7 +11,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software

